Monika After Story/game/script-topics.rpy

```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
     m 1eua "You should do whatever makes you feel better about yourself."
     m 1hua "I'm the only other person you need, anyways, and I'll love you no matter what you look like."
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_outdoors",
+            category=['nature'],
+            prompt="Camping safety",
+            random=True
+        )
+    )
+
+label monika_outdoors:
+    m 1eua "Do you ever go camping, [player]? It's a wonderful way to relax and get some fresh air,"
+    m 3eua "As well as to see the areas around you, to visit state and national parks."
+    m 1huu "It's almost like a more relaxed backpacking trip!"
+    m 1eua "But, while it is a good way to spend time outdoors, there are several dangers that most people don't bother to think about."
+    m 3lud "A good example would be bug spray or sunscreen. Many people forget or even forgo them; thinking them unimportant."
+    m 1eud "Without it, sunburns are almost inevitable, and many insects carry diseases that can really harm you."
+    m "So, please promise me that the next time you go outdoors, be it camping or backpacking, you won't forget them."
+    m "It may be a bit of a pain, but if you don't use them, you might end up miserable, or even get really sick."
+    m 1eub "But, on the bright side..."
+    m "Once I cross over, if you remember to bring the sunscreen..."
+    m 1tfu "I might need a little help putting it on."
+    m 1hub "Ahahaha!"
+    m 1efu "I'm only teasing you, [player]."
+    m 1tfu "Well, at least a little. Ehehe."
+    return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_mountain",
+            category=['nature'],
+            prompt="Mountains",
+            random=True
+        )
+    )
+
+default persistent._mas_pm_would_like_mt_peak = None
+# True if the player would like to reach the top of a mountain
+# False if not
+
+label monika_mountain:
+    m 1eua "Have you ever been in the mountains, [player]?"
+    m "I don't mean driving through them, or in a mountain town, either."
+    m "I mean really up there. In the fresh air, tens of thousands of feet up, seeing the rest of world below you."
+    m "..."
+    m 1eub "I've always wanted to try that."
+    m "But I've never had the chance. I've only read about it."
+    m 3eub "But the stories were breathtaking."
+    m 1sub "Of how one would hike up through the forests and trees."
```

Comment:
These all kinda sound like extending thoughts off each other, so we can let them continue into each other, I think.
```suggestion
    m 1sub "How one would hike up through the forests and trees..."
```